.github/workflows/stage-build.yml

```diff
@@ -0,0 +1,24 @@
+### This is the Terraform-generated dev-build.yml workflow for the asati-stage app repository ###
+### If this is a Lambda repo, uncomment the FUNCTION line at the end of the document ###
+### If the container requires any additional pre-build commands, uncomment and edit ###
+### the PREBUILD line at the end of the document. ###
+name: Stage Container Build and Deploy
+on:
+    workflow_dispatch:
+    pull_request:
+        branches:
+            - main
+        paths-ignore:
+            - '.github/**'
+
+jobs:
+    deploy:
+        name: Dev Container Deploy
```

Comment:
Should this be `Stage Container Deploy`?

Reply:
opps! I pushed up a commit to correct that. Apologies, for the oversight.

If you refresh your review, I will go ahead and merge this with `main` and we can do some basic deploy/testing and close the rest of the tickets (and the infra automation sub-project related to this project). I will ping you on Slack after you do the review and I do the merge.